spec/spec_helper.rb

```diff
@@ -33,11 +33,10 @@
   raise <<-MESSAGE
     No schemas found
 
-    The Publishing API tests require that the govuk-content-schemas
+    The Publishing API tests require that content schemas
     are available. These are accessed through the GovukSchemas gem,
```

Comment:
Whilst you're editing this file, could `GovukSchemas` be changed to `govuk_schemas` to actually match the name of the gem.

Reply:
Good point, have opted to remove this message as per Kevin's suggestion below